htlcswitch/link.go

```diff
@@ -1306,6 +1307,20 @@ func (l *channelLink) htlcManager() {
 		case <-l.quit:
 			return
 		}
+
+		// After we are finished processing the event, if the link is
```

Comment:
So this won't always be reached, certain cases above can cause us to `continue` to the next evaluation of the loop. Eg: fee update ticks, but nothing actually needed. 

I think in practice though, this'll eventually be reached as anytime we process an actual update, we'll drop down there.